tree-sitter-c-sharp/src/parser.c:534:32: error:      warning: trigraph ??= ignored, use -trigraphs to enable [-Wtrigraphs]

```
[2 of 2] Compiling TreeSitter.CSharp ( TreeSitter/CSharp.hs, /home/runner/work/haskell-tree-sitter/haskell-tree-sitter/dist-newstyle/build/x86_64-linux/ghc-8.8.1/tree-sitter-c-sharp-0.1.0.0/noopt/build/TreeSitter/CSharp.o )

vendor/tree-sitter-c-sharp/src/parser.c:534:32: error:
     warning: trigraph ??= ignored, use -trigraphs to enable [-Wtrigraphs]
       [anon_sym_QMARK_QMARK_EQ] = "??=",
                                    ^
    |
534 |   [anon_sym_QMARK_QMARK_EQ] = "??=",
    |                                ^

vendor/tree-sitter-c-sharp/src/parser.c:534:32: error:
     warning: trigraph ??= ignored, use -trigraphs to enable [-Wtrigraphs]
       [anon_sym_QMARK_QMARK_EQ] = "??=",
                                    ^
    |
534 |   [anon_sym_QMARK_QMARK_EQ] = "??=",
    |                                ^
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -8,8 +8,8 @@ jobs:
     runs-on: ubuntu-16.04
     strategy:
       matrix:
-        ghc: ["8.6.5", "8.8.1"]
-        cabal: ["3.0"]
+        ghc: ["8.6.5", "8.8.3"]
+        cabal: ["3.2.0.0"]
 
     steps:
     - uses: actions/checkout@master

--- a/tree-sitter-c-sharp/tree-sitter-c-sharp.cabal
+++ b/tree-sitter-c-sharp/tree-sitter-c-sharp.cabal
@@ -43,7 +43,7 @@ library
   install-includes:    tree_sitter/parser.h
   c-sources:           vendor/tree-sitter-c-sharp/src/parser.c
   extra-libraries:     stdc++
-  cc-options:          -fno-trigraphs -Wno-trigraphs
+  cc-options:          -Wno-trigraphs
 
 source-repository head
   type:     git

--- a/tree-sitter-c-sharp/tree-sitter-c-sharp.cabal
+++ b/tree-sitter-c-sharp/tree-sitter-c-sharp.cabal
@@ -43,6 +43,7 @@ library
   install-includes:    tree_sitter/parser.h
   c-sources:           vendor/tree-sitter-c-sharp/src/parser.c
   extra-libraries:     stdc++
+  cc-options:          -fno-trigraphs -Wno-trigraphs
 
 source-repository head
   type:     git

